### ISSUE TITLE
Comment: Remote KeepassXC "access/update" using X-Windows

### ISSUE THREAD

**Issue by sainokawara** [keepassxreboot/keepassxc#2360]
When the KeepassXC database resides on a Linux system, in my opinion, the best method of "accessing/updating" it from remote systems is by using X-Windows, rather than KeepassXC-Browser.  I use this technique from my other Linux and Windows systems and it works very well.  Perhaps, someone in your group could provide a writeup on how to do this on your website for the more technical users?  There are a number of products providing access to Linux X-Windows systems from Windows including MobaXterm which is free for personal use.  Just a thought.

**Comment by droidmonkey** [comment]
Huh? We already have auto-type that uses the X-Windows system to type in usernames and passwords.

**Comment by phoerious** [comment]
I don't think we want to have that as an officially supported method. X over network is something that used to work 20 years ago, but is now one of the most inefficient remote protocols.

**Comment by sainokawara** [comment]
I doubt that most users even understand the discussion so there is probably little demand.  However, my home Intranet is 1 Gb/s and I am the only person using it so efficiency is not an issue.  I am more concerned with convenience and paranoid about security.  And, anything that even gets close to JavaScript is suspect in my mind.  I keep my Browsers locked-down as tight as I can.

**Comment by phoerious** [comment]
If you are concerned of security, X11 is the last thing you should be using.
